operations/mimir/autoscaling.libsonnet

```diff
@@ -80,7 +80,9 @@
             //
             // Read more:
             // https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#algorithm-details
-            threshold: trigger.threshold,
+            // 
+            // We also have to ensure that the threshold is an integer
+            threshold: std.split(trigger.threshold, '.')[0],
```

Comment:
@pracucci what do you think of this solution, i.e. to ensure an integer when configuring the KEDA threshold?

Reply:
Thinking about it, what if instead we had jsonnet raise an error if `trigger.threshold` is not an integer?